source/_patterns/components/04-toast-notifications/00-toast-notification.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="pf-toast alert {{ alertClass }} {{#if alertDismissible}}alert-dismissible{{/if}}" role="alert">
```

Comment:
What do you think about adding a comment at the top of the .hbs with all the handlebars variables that configure the pattern?